roocs_grids/scripts/create_grid_file.py

```diff
@@ -2,6 +2,9 @@
 from pathlib import Path
 
 import xarray as xr
+
+# FIXME: This is a circular dependency, as clisops will be listing roocs-grids as a dependency
```

Comment:
The `scripts` directory is basically a directory with auxiliary scripts to prepare the grid files. None of its contents are imported by the roocs_grids module. I don't know if there is a proper approach for something like this. Maybe one could rename it to `aux` and move it one directory up, next to `tests`, `roocs_grids` so it won't get accidentally imported?